.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+<!--
+This issues database is used to keep track of bugs and new features.
+For questions and support, please visit the Gitter channel instead:
+  https://gitter.im/Vincit/objection.js
+-->
+
+# Description/Steps to reproduce
+
+<!--
+If a feature: A description of the feature.
+If a bug: The steps to reproduce the issue.
+-->
+
+# Link to reproduction test-case
+
+<!--
+Please link to a test-case for reproduction, ideally as a repository with a
+`packages.json` that installs all required dependencies to reduce confusion.
```

Comment:
Also a test case for objection's test suite would be execellent and a standalone file is fine. runkit is also a great option.

Reply:
I was thinking that a reproduction as a repository is probably too much to ask and people would just forget the bug and work around it most of the time. We could add a standalone file template to the project that can be copy-pasted and with small modifications, the bug could be reproduced?

Reply:
That sounds like a good idea. I've never used runkit. Would you like to prepare this standalone file?

Reply:
Sure, I'll write the file.